### ISSUE TITLE
Disable/Enable Eye Tracking in App

### ISSUE THREAD

**Issue by austinbhale** [StereoKit/StereoKit#527]
## Describe the feature
Would it be possible to enable eye tracking manually? I have a HoloLens 2 application where I want to request permissions at the app's startup but I don't want to accidentally request a permission while the app asks for the eye tracking permission, which might result in undefined behavior. It also might be useful for developers to notify users whether the eye tracking permission is enabled via a simple `get => Input.EyeTrackingEnabled`, which is different than seeing if it is supported by the headset like here: https://learn.microsoft.com/en-us/uwp/api/windows.perception.people.eyespose.issupported?view=winrt-22621

## An example use-case
The ability to turn on/off eye tracking (or other user input) can be useful for many applications. There could be a `SKSetting` called `disable_eye_tracking` to disable it initially and then the dev could enable/disable it within the application. Something like `Input.EnableEyeTracking()` calls the necessary OpenXR functions to enable/disable it in the app. Do you think this is a reasonable request?

**Comment by maluoi** [comment]
Yeah, there's a new collection of permission based features on Quest Pro as well! With that in mind I've been thinking it'd be very helpful/important to have some easy tools for requesting feature permissions that's cross-platform and managed from a central location.

So yeah, super valid ask!
